packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -425,14 +421,26 @@ async function createFile(entry, options, method, fileId) {
 
   let fileDocument
   if (method === 'create') {
-    fileDocument = await cozy.files.create(toCreate, createFileOptions)
+    const toSave = {
+      _type: 'io.cozy.files',
```

Comment:
you have created a `FILES_DOCTYPE` you should use it. Or if not, you can remove it.